src/components/ec-phone-number-input/ec-phone-number-input.vue

```diff
@@ -333,12 +339,11 @@ export default {
       this.$emit('phone-number-change', evt);
     },
     getCountryFlagPath(countryCode) {
-      if (!countryCode) {
+      if (!countryCode || !Object.keys(countries).includes(countryCode)) {
```

Comment:
This `getCountryFlagPath` function is called in a for loop, I think we can optimise this a bit. The `countries` object is not going to change so we can create a Set outside of the component:
```
import countries from 'svg-country-flags/countries.json';
const supportedCountries = new Set(Object.keys(countries));
```

... and then just call `if (!countryCode || !supportedCountries.has(countryCode)) {`